### ISSUE TITLE
Regulations page design updates - new intro language, remove "other formats"

### ISSUE THREAD

**Issue by isaacjamesbaker** [fecgov/fec-cms#6393]
### Summary

**What we're after:**
In a prior ticket we uploaded1977-1996 CFRs into our CMS and linked on the Regulations. There are some further proposed design updates related to this page that Jonella has mocked up. We need to follow up with OGC to confirm the language is correct, the design makes sense, and then once approved, we need to publish the updated page.

### Related issues
This ticket is a continuation of a prior closed ticket [6917](https://github.com/fecgov/fec-cms/issues/6197)

### Completion criteria
- [x] New opening section that includes draft explanatory language letting people know FEC relies on eCFR, anything else needed to say about search and the search regulations button (waiting on OGC input)
- [x] eCFR link is moved to our explanatory text
- [x] The "other formats" links on the sidebar are removed since they are now superfluous.

Mockup from Jonella 
![Regulations-redesign](https://github.com/fecgov/fec-cms/assets/31663028/5be8bb36-212a-4671-90c5-8bda69ef85ac)

### Future work
Isaac will update OGC once the page has been approved and published.
